Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
-    <string name="could_not_start_saving_no_permissions">Could not start downloading images, no WRITE_ACCESS permission was granted</string>
-    <string name="update_manager_copy_apk_title">Copy apk?</string>
-    <string name="update_manager_copy_apk">Would you like to store this apk?</string>
-    <string name="update_manager_could_not_convert_uri">Couldn\'t convert uri (%1$s) into an ExternalFile</string>
-    <string name="update_manager_input_file_does_not_exist">Input apk file (%1$s) does not exist</string>
+    <string name="media_settings_could_not_create_default_baseDir">Could not create default base directory because %1$s</string>
+
+    <string name="site_uses_dynamic_boards">"Since the site likely has dynamic boards you must add one from the \"Boards\" menu first."</string>
+    <string name="could_not_start_saving_no_permissions">Grant write access permission to start downloading images</string>
+    <string name="update_manager_copy_apk_title">Copy APK?</string>
+    <string name="update_manager_copy_apk">Would you like to store this APK?</string>
+    <string name="update_manager_could_not_convert_uri">Could not convert URI (%1$s) into an ExternalFile</string>
+    <string name="update_manager_input_file_does_not_exist">Input APK file (%1$s) does not exist</string>
     <string name="update_manager_output_file_does_not_exist">Output file (%1$s) does not exist</string>
-    <string name="update_manager_could_not_copy_apk">Couldn\'t copy file contents from file (%1$s) into (%2$s)</string>
-    <string name="update_manager_apk_copied">Apk successfully copied</string>
-    <string name="settings_show_copy_apk_dialog_title">Show copy apk dialog when downloading an update</string>
-    <string name="settings_show_copy_apk_dialog">Every time you download a new update a dialog with suggestion to copy that apk to some other directory will be shown</string>
+    <string name="update_manager_could_not_copy_apk">Could not copy file contents from file (%1$s) into (%2$s)</string>
+    <string name="update_manager_apk_copied">APK copied</string>
+    <string name="settings_show_copy_apk_dialog_title">Show \"copy APK\" dialog when downloading an update</string>
+    <string name="settings_show_copy_apk_dialog">Every time you download a new update a dialog with suggestion to copy that APK to some other directory will be shown</string>
 
-    <string name="could_not_initialized_captcha">Couldn\'t initialize captcha, reason: %1$s</string>
+    <string name="could_not_initialized_captcha">Could not present CAPTCHA because %1$s</string>
     <string name="fail_reason_webview_is_not_installed">WebView is not installed</string>
-    <string name="fail_reason_some_part_of_webview_not_initialized">Some part of WebViewChromium couldn\'t get initialized: (%1$s)</string>
+    <string name="fail_reason_some_part_of_webview_not_initialized">Could not start some part of WebViewChromium because %1$s</string>
```

Comment:
Would suggest
``Unable to initialize WebViewChromium because %1$s``

Reply:
Will revise without "part of"